File: IDSFintechPortal.Api/Models/Document.cs
namespace IDSFintechPortal.Api.Models
{
    public class Document
    {
        public int Id { get; set; }
        public string DocumentName { get; set; } = string.Empty;
        public string? DocumentType { get; set; }
        public int ProductId { get; set; }
        public string? Description { get; set; }
        public string? FileReference { get; set; }
        public DateOnly? LastUpdatedDate { get; set; }
    }
}

File: ids-fintech-portal-frontend/src/types/Document.ts
export interface Document {
    id: number;
    documentName: string;
    documentType: string | null;
    productId: number;
    description: string | null;
    fileReference: string | null;
    lastUpdatedDate: string | null;
}

export interface CreateDocumentDto {
    documentName: string;
    documentType?: string;
    productId: number;
    description?: string;
    fileReference?: string;
    lastUpdatedDate?: string;
}
